frontend/src/components/LandingPage.tsx

```diff
@@ -25,13 +25,13 @@ const LandingPage = () => {
         }}>
             <LandingPageHero />
             <LandingPageFeatureElections electionIds={[
-                '52c2b793-bdfe-49e8-b4cf-f448108c39af',
-                'e45e93b1-9e4e-4a87-bb80-e7dbbd6e4721',
-                '5c84e93b-b01c-4ab0-82ee-7181cb7ac995'
-            ]} />
-            <LandingPageFeatures />
+                '658043c3-27a6-4944-802e-1dfc8825a6dd',
```

Comment:
These might be better as env variables so they're easier to change and can be different for dev and prod.

Reply:
Makes sense, I moved it to a comma separated env variable